FRINXio/frinx-frontend
# handle expired unistore transaction

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FD-674` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  |


HOW TO TEST:

- go to l3vpn and create site (filling name in the site form should be enough)
- open developer tools and save UNICONFIGTXID cookie value in clipboard
- discard changes (now you should see new UNICONFIGTXID)
- replace new UNICONFIGTXID with old one from clipboard.

now you have invalid transaction id and when you refresh page you should obtain modal, that transaction is invalid and once you click ok, you should obtain new one
